docs/app/Examples/collections/Table/Types/TableApprove.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Button, Checkbox, Icon, Table } from 'stardust'
+
+const TableApprove = () => {
+  return (
+    <Table compact celled definition>
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell />
+          <Table.HeaderCell>Name</Table.HeaderCell>
+          <Table.HeaderCell>Registration Date</Table.HeaderCell>
+          <Table.HeaderCell>E-mail address</Table.HeaderCell>
+          <Table.HeaderCell>Premium Plan</Table.HeaderCell>
+        </Table.Row>
+      </Table.Header>
+
+      <Table.Body>
+        <Table.Row>
+          <Table.Cell collapsing>
+            <Checkbox fitted slider />
```

Comment:
Noted in `TableFullWidth.js`, the [`fitted` prop is auto added](http://localhost:8080/modules/checkbox#fitted) when there is no `label` defined.